os-lifecycle-policy.md

```diff
@@ -2,18 +2,20 @@
 
 .NET is [supported by Microsoft](microsoft-support.md) on multiple operating systems, defined per .NET version.
 
-Each supported operating system has a lifecycle defined by its sponsor organization (for example, Microsoft, Red Hat, Debian, or Apple). The .NET team applies each of those lifecycle schedules to inform adding and removing support for operating system versions.
+A more restrictive policy is used for [.NET container images](https://github.com/dotnet/dotnet-docker/blob/main/documentation/supported-platforms.md).
 
-When an operating system version goes out of "mainstream" or "free" support, we stop testing that version and providing support for that version. This means that users will need to move forward to a supported operating system version to get support.
+## Supported .NET versions
 
-The following documents define currently supported operating system versions for supported .NET Core versions:
+The following documents describe supported operating systems for .NET:
 
-* [.NET 6 supported OS versions](release-notes/6.0/6.0-supported-os.md)
+* [.NET 6 supported OS versions](release-notes/6.0/supported-os.md)
 * [.NET 5 supported OS versions](release-notes/5.0/5.0-supported-os.md)
 * [.NET Core 3.1 supported OS versions](release-notes/3.1/3.1-supported-os.md)
 * [.NET Core 2.1 supported OS versions](release-notes/2.1/2.1-supported-os.md)
 
-The following documents define (historical) operating system support for out-of-support .NET Core versions:
+## End-of-life .NET versions
```

Comment:
We're using the term out of support versions on pages like https://dotnet.microsoft.com/download/dotnet

Reply:
Fixed